ModelBundle/DataFixtures/MongoDB/LoadNodeDemoData.php

```diff
@@ -26,6 +26,36 @@ class LoadNodeDemoData extends AbstractFixture implements OrderedFixtureInterfac
      */
     public function load(ObjectManager $manager)
     {
+
+        $manager->getConnection()->initialize();
+        $dataBase = $manager->getDocumentDatabase('OpenOrchestra\ModelBundle\Document\Node');
+        $dataBase->execute("
+db.node.createIndex( { nodeId: 1, siteId: 1, language: 1, version: 1 }, { unique: true } )
+db.system.js.save(
+    {
+        _id : 'duplicateNode' ,
+        value : function (data){
+            var node = null
+            while (1) {
+                var cursor = db.node.find( { nodeId: data.nodeId, siteId: data.siteId, language: data.language }).sort( { version: -1 } ).limit(1);
+                node = cursor.next()
+                delete node._id;
+                node.version = node.version + 1;
```

Comment:
This function is like a proc stock. So if you change the cast in ORM, you have to change proc stock. I think we could keep this logic in ODM? @itkg-ngilain